unsafe/src/main/java/org/apache/spark/unsafe/bitset/BitSetMethods.java

```diff
@@ -68,6 +68,19 @@ public static boolean isSet(Object baseObject, long baseOffset, int index) {
   }
 
   /**
+   * Returns {@code true} if all bits are set.
+   */
+  public static boolean allSet(Object baseObject, long baseOffset, long bitSetWidthInWords) {
+    long addr = baseOffset;
+    for (int i = 0; i < bitSetWidthInWords; i++, addr += WORD_SIZE) {
+      if (Platform.getLong(baseObject, addr) == 0) {
```

Comment:
This confuses me. It seems that you are checking that all bits in a given word are not set (0), instead of checking that any one the bits is not set. Am I missing something?